packages/settings-library/src/settings_library/comp_services.py

```diff
@@ -0,0 +1,25 @@
+from pydantic import ByteSize, NonNegativeInt, validator
+from settings_library.base import BaseCustomSettings
+
+_DEFAULT_MAX_NANO_CPUS_VALUE = 4 * pow(10, 9)
+_DEFAULT_MAX_MEMORY_VALUE = 2 * pow(1024, 3)  # 2 GiB
+
+
+class CompServices(BaseCustomSettings):
+    DEFAULT_MAX_NANO_CPUS: NonNegativeInt = _DEFAULT_MAX_NANO_CPUS_VALUE
```

Comment:
NonNegative? can you really set DEFAULT_MAX_NANO_CPUS=0??

Reply:
yes, then it will go to default.